op-tests/test-modpow.txt

```diff
@@ -0,0 +1,82 @@
+% 5267150965625435864212627812265569613424934328821773337492 697765432255771648207431025164631109816514495524815416162 => 382792939835034326760610636113151844709332860148065424358 | 1424
+modpow 5323234311971891384074940450157276565325739709920219872256 5393587130001156103739338371876848805375529236699906826621 1765717912111822190672034850899428474523885890492339984895 => 1474178189165936573164291011565018377626176030901287229811 | 18098
+% 5323234311971891384074940450157276565325739709920219872256 1765717912111822190672034850899428474523885890492339984895 => 26080575636424812058835897458991141754082038443199917571 | 1424
+modpow 586918905347958651527374084101851822104720155164102870676 957519517681344878843393151150797049627091951639782539434 5588839971169877365798072413857127472099405929374710467767 => 3695379230730624502645402080869683287220511789290550257948 | 18338
+% 586918905347958651527374084101851822104720155164102870676 5588839971169877365798072413857127472099405929374710467767 => 586918905347958651527374084101851822104720155164102870676 | 1424
+modpow 2535072461212885977356432886030417155554351934737467955782 217851043345470355834534715842769407245179863021057559724 6081638570091334915579467683836944866341382957565643910571 => 5216353536297671276218816266070693681202106241194420294777 | 18338
+% 2535072461212885977356432886030417155554351934737467955782 6081638570091334915579467683836944866341382957565643910571 => 2535072461212885977356432886030417155554351934737467955782 | 1424
+modpow 379381184782583886160798882693322550895543756104221513547 915671330163880449472931063032573220285863459038113665044 4827293201730901888701785199482402666774902234258786888061 => 3905408349744560216995403352120706665260520665673380270199 | 18338
+% 379381184782583886160798882693322550895543756104221513547 4827293201730901888701785199482402666774902234258786888061 => 379381184782583886160798882693322550895543756104221513547 | 1424
+modpow 3195368787213672865566401839626864639167178250334332902956 73504708037419427367735470928041825274387433527777430073 4811637027825861720291821784099893200774466654637793791600 => 3475690977553257351143127975640803368777626976201134848416 | 18353
+% 3195368787213672865566401839626864639167178250334332902956 4811637027825861720291821784099893200774466654637793791600 => 3195368787213672865566401839626864639167178250334332902956 | 1438
+modpow 3393328799375776415256184972208980097810780581787744702798 3052118175933073395388339784015782524774488551946159481690 5436126395750286638399600726623817593321195552838489065236 => 4440460779126825370214984702000853599873253048512412194128 | 18353
+% 3393328799375776415256184972208980097810780581787744702798 5436126395750286638399600726623817593321195552838489065236 => 3393328799375776415256184972208980097810780581787744702798 | 1438
```

Comment:
A lot of these tests are of the form `(% A B)` where `A < B`, which isn't very interesting. Maybe add some where `B` is 128 bits instead of 256?

Reply:
yeah, I would expect about half of them are like that. I'll add a few more cases